app/models/discovery_attribute_set.rb

```diff
@@ -3,5 +3,4 @@ class DiscoveryAttributeSet < ApplicationRecord
 
   validates :cpu_count, :presence => true, :numericality => {:greater_than_or_equal_to => 0}
   validates :memory, :presence => true,    :numericality => {:greater_than_or_equal_to => 0}
-  validates :host, :presence => true
```

Comment:
I think we should try to keep this, see my earlier comment for a possible solution.

Reply:
There are many other reasons why discovery process can fail and if there is no attribute associated the discover host gets stuck. Any attempt to save it lead to validation error. And since I am removing the other side of the validation, I see no particular reason why to have it here.

Reply:
I can probably keep this one, it's not causing any harm it's the other one which breaks re-discovery when attribute_set related table was not associated due to bug/problem.

Reply:
Replaced with `validates_associated` on the other side of the association.